tox.ini

```diff
@@ -23,7 +23,7 @@ deps =
     -r{toxinidir}/test-requirements.txt
 commands =
     coverage erase
-    stestr run --serial --test-path {[testenv]unit_tests} {posargs}
+    stestr run --random --slowest --test-path {[testenv]unit_tests} {posargs}
```

Comment:
If parallel breaks something we can always serialize a class (i think).

Why the --randomize? I don't see that used upstream, it's not even in the help message for stestr

Reply:
That flag is described here: <https://stestr.readthedocs.io/en/latest/MANUAL.html#cmdoption-stestr-run-random>

> --random
>   Randomize the test order after they are partitioned into separate workers

Reply:
@brianphaley "--random" is there to ensure that the unit test cases don't have hidden dependencies between them. For example, randomization would reveal if test case "a" is setting a global state and test case "b" is unknowingly using it and is passing because of it. If our unit cases are indeed "atomic" and "isolated" (as they should be), the order of execution shouldn't matter.